slidedeck/src/slides/01-layout-tools/01-create-react-app.pug

```diff
@@ -8,9 +8,12 @@ section
   ul
     li Jumpstart React app with no configuration
     li Opinionated setup of Webpack and Babel
-      li React, JSX, ES6, CSS, Jest Testing
+      ul
+        li React, JSX, ES6, CSS, Jest Testing
     li Allows for easy gradual adoption
-    li `Eject` to export custom configuration
+    li
+      strong  'Eject'
+      |  to export custom configuration
```

Comment:
This something intentional?

Reply:
yeah, makes 'Eject' stand out, like we do on previous slides with commands